packages/headless/src/features/analytics/search-analytics-actions-loader.ts

```diff
@@ -725,6 +727,19 @@ export interface SearchAnalyticsActionCreators {
     void,
     AsyncThunkInsightAnalyticsOptions<StateNeededBySearchAnalyticsProvider>
   >;
+  /**
+   * The event to log when the Copy To Clipboard result action is clicked.
+   *
+   * @param result - The result.
+   * @returns A dispatchable action.
+   */
+  logCopyToClipboard(result: Result): AsyncThunkAction<
```

Comment:
Do we really need/want to also re-export it here ? The concept of "result action" seems only applicable for the insight panel use case.

Reply:
indeed! I've just re-checked with the Quantic team, probably we will need this feature for the search use case in the future but for now we only need it for the insight use case, so it would make sense to only add this logic for the insight use case especially that it's easy to add the logic for search when needed.
I'll update my PR accordingly.